frontend/src/components/my-grant-profile-page-handler/my-grant.tsx

```diff
@@ -0,0 +1,225 @@
+import {
+  Button,
+  Grid,
+  GridColumn,
+  Link,
+  Spacer,
+  Text,
+  VerticalStack,
+} from "@python-italia/pycon-styleguide";
+import { FormattedMessage } from "react-intl";
+
+import { useCurrentLanguage } from "~/locale/context";
+import { DeadlineStatus, Status as GrantStatus } from "~/types";
+import type { MyProfileWithGrantQuery } from "~/types";
+
+import { useCountries } from "~/helpers/use-countries";
+import { createHref } from "../link";
+import { Sidebar } from "./sidebar";
+
+type Props = {
+  grant: MyProfileWithGrantQuery["me"]["grant"];
+  deadline: MyProfileWithGrantQuery["conference"]["deadline"];
+};
+
+export const MyGrant = ({ grant, deadline }: Props) => {
+  const language = useCurrentLanguage();
+  const countries = useCountries();
+
+  const canManageGrant = [
+    GrantStatus.WaitingForConfirmation,
+    GrantStatus.Confirmed,
+    GrantStatus.WaitingList,
+    GrantStatus.WaitingListMaybe,
+  ].includes(grant.status);
+
+  const getCountryLabel = (value: string): string | undefined => {
```

Comment:
we should define this function outside and send countries as param, especially if it could be used in other places

Reply:
I moved this in the `helpers` and I called `country-utils`... better names are welcomed